kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -898,6 +927,180 @@ struct ResetMetadata {
   }
 };
 
+struct ListColumnFamilies {
+  const bool read_only = true;
+  std::string description() const {
+    return "listColumnFamilies\n"
+           " List column families in Rocksdb";
+  }
+
+  std::string toJson(const std::unordered_set<std::string> &result) const {
+    std::ostringstream oss;
+    oss << "{\n\t\"column_families\":[";
+    if (!result.empty()) {
+      auto it = result.begin();
+      oss << "\"" << *it << "\"";
+      while (++it != result.end()) {
+        oss << ",\n\"" << *it << "\"";
+      }
+    }
+    oss << "]\n}";
+    return oss.str();
+  }
+
+  std::string execute(const KeyValueBlockchain &adapter, const CommandArguments &args) const {
+    auto result = adapter.db()->columnFamilies();
+    return toJson(result);
+  }
+};
+
+struct GetColumnFamiliesStats {
+  const bool read_only = true;
+  const std::string *properties[4] = {&rocksdb::DB::Properties::kNumLiveVersions,
+                                      &rocksdb::DB::Properties::kTotalSstFilesSize,
+                                      &rocksdb::DB::Properties::kEstimateNumKeys,
+                                      &rocksdb::DB::Properties::kNumEntriesActiveMemTable};
+  std::string description() const {
+    std::ostringstream oss;
+    oss << "GetColumnStats optional:COLUMN_FAMILY_NAME\n"
```

Comment:
Consider naming it in the same way as the command itself. We could call it `GetColumnFamilyStats` (in singular) as we might only request one column family. Also, in the interest of doing what other commands do, you can use `GetColumnFamilyStats [COLUMN_FAMILY_NAME]` to denote it is an optional parameter.

Reply:
done